sdks/python/apache_beam/yaml/generate_yaml_docs.py

```diff
@@ -193,14 +200,19 @@ def io_grouping_key(transform_name):
 # Exclude providers
 SKIP = {}
 
+def normalize_beam_version(desc):
+  return desc.replace(
+      'BEAM_VERSION', 'current' if '.dev' in beam_version else beam_version)
```

Comment:
I think it's preferable to leave the .dev in place.

Reply:
Sure, they are broken links anyway, so either way, they don't point to the expected doc